components/event-receiver/org.wso2.carbon.event.receiver.core/src/main/java/org/wso2/carbon/event/receiver/core/internal/type/map/MapInputMapper.java

```diff
@@ -82,7 +87,11 @@ public Object convertToMappedInputEvent(Object obj) throws EventReceiverProcessi
             Map eventMap = (Map) obj;
             List<Object> outObjList = new ArrayList<Object>();
             for (int i = 0; i < this.attributePositionKeyMap.length; i++) {
-                outObjList.add(eventMap.get(this.attributePositionKeyMap[i]));
+                if (eventMap.get(this.attributePositionKeyMap[i])==null) {
```

Comment:
@minudika the formatting in the above line seems incorrect. Can you reformat this file using your IDE and push that...